<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupLogoDialog</class>
 <widget class="QWidget" name="StartupLogoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="1" column="2">
    <widget class="QLabel" name="m_TransparentBeta">
     <property name="text">
      <string>BETA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="m_TransparentVersion">
     <property name="text">
      <string>Development version</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="m_TransparentConfidential">
     <property name="text">
      <string>Specified in the StartupLogoDialog constructor.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="m_TransparentText">
     <property name="text">
      <string>Starting Editor...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>581</width>
       <height>176</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
